scipy/stats/tests/test_stats.py

```diff
@@ -3319,6 +3329,14 @@ def test_ttest_rel():
     assert_array_almost_equal(np.abs(p), pr)
     assert_equal(t.shape, (3, 2))
 
+    # test alternative parameter
+    assert_raises(ValueError, stats.ttest_rel, rvs1, rvs2, alternative="error")
+
+    t,p = stats.ttest_rel(rvs1, rvs2, axis=0, alternative="less")
+    assert_array_almost_equal([t,p],(tr,1 - pr/2))
```

Comment:
Note above:
```
    assert_array_almost_equal(np.abs(t), tr)
    assert_array_almost_equal(np.abs(p), pr)
```
This is pretty standard in our tests. Please separate these.
If you were to combine them:
```suggestion
    assert_array_almost_equal((t,p),(tr,1 - pr/2))
```

Reply:
I'll look at the rest later.